### ISSUE TITLE
Samsung Odyssey Plus (WMR) ingame will not center if OpenVR-AdvancedSettings is installed

### ISSUE THREAD

**Issue by horst789github** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#269]
**Describe the bug**
Samsung Odyssey Plus (WMR) ingame will not center if OpenVR-AdvancedSettings is installed:
e.g. the menu of various games is in the ground and cannot be centered with the new centering, it immediately jumps back again. this only occurs if OpenVR-AdvancedSettings is installed. As soon as you uninstall it, you can re-center everything correctly and play games wonderfully.
https://us.community.samsung.com/t5/Monitors-and-Memory/Samsung-Odyssey-Plus-ingame-menus-will-not-center/td-p/637872/page/2

**To Reproduce**
install OpenVR-AdvancedSettings > start WMR > SteamVR > game > not centered game > no recenter posible > uninstall OpenVR-AdvancedSettings > now recenter is posible

**Expected behavior**
fix this bug, i want to use this nice addon with wmr

**Software (please complete the following information):**
 - OS: Win10 Pro 1903
 - Advanced Settings Version: 3.2.0

**Hardware (please complete the following information):**
 - CPU: Intel I7 8700k
 - Graphics Card: RTX2080ti
 - HMD: Samsung Odyssey Plus (WMR)

**Comment by Kung-vr** [comment]
Thanks for the report!

There's some buggy behavior right now with WMR if it's not set up for "all experiences" and with the boundary on. Running in that specific setup might might fix the issue for you. If it's with seated mode games though that could be another issue. Some games work and some don't right now. This is an ongoing collection of issues with inconsistencies for how different games interact with the chaperone and universe center.

I'm thinking our best bet might be to add some debug options for different combinations of old and new motion styles. I'll keep that in mind as something to add.

**Comment by 2m** [comment]
Do you know if there is a way to hide the WMR boundary and still keep reset functionality?

**Comment by fvdheide1963** [comment]
> **Describe the bug**
> Samsung Odyssey Plus (WMR) ingame will not center if OpenVR-AdvancedSettings is installed:
> e.g. the menu of various games is in the ground and cannot be centered with the new centering, it immediately jumps back again. this only occurs if OpenVR-AdvancedSettings is installed. As soon as you uninstall it, you can re-center everything correctly and play games wonderfully.
> https://us.community.samsung.com/t5/Monitors-and-Memory/Samsung-Odyssey-Plus-ingame-menus-will-not-center/td-p/637872/page/2
> 
> **To Reproduce**
> install OpenVR-AdvancedSettings > start WMR > SteamVR > game > not centered game > no recenter posible > uninstall OpenVR-AdvancedSettings > now recenter is posible
> 
> **Expected behavior**
> fix this bug, i want to use this nice addon with wmr
> 
> **Software (please complete the following information):**
> 
> * OS: Win10 Pro 1903
> * Advanced Settings Version: 3.2.0
> 
> **Hardware (please complete the following information):**
> 
> * CPU: Intel I7 8700k
> * Graphics Card: RTX2080ti
> * HMD: Samsung Odyssey Plus (WMR)

I have the exact same issue with WMR in combination with OpenVR-AdvancedSettings. Recentering works in Assetto Corsa and Rfactor2, but in iRacing it doesn't. Has me stuck in a weird seating position when i launch the game and no way to get it corrected. I mapped a key to Recenter HMD, but it simply doesn't work. So for now I need to de-install OpenVR-AdvancedSettings. I really wish to keep using it, so I hope this issue will be fixed asap. My HMD is a HP Reverb by the way.

Question, how to de-install OpenVR-AdvancedSettings?

**Comment by ykeara** [comment]
@fvdheide1963 you can in its settings menu turn off "auto-start" and it shouldn't start unless you manually run it, but you can run the un-installer located @ C:\Program Files\OpenVR-AdvancedSettings, manually deleting it will also work, but you should run it with `--force-remove-manifest`  before manually deleting [uninstaller does this step for you.]

**Comment by ykeara** [comment]
Duplicate of #170 


Just trying to keep everything centralized if issue is the same.
